src/components/Form.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+
+class Form extends Component {
+  state = {
+    id: Math.ceil(Math.random() * 100000),
+    name: String(),
+    description: String(),
+    inCart: false
```

Comment:
`id` and `inCart` shouldn't be part of the form. It has nothing to do with this component, and it definitely shouldn't be in state because it never gets changed.